homeassistant/components/matter/entity.py

```diff
@@ -86,6 +88,9 @@ async def async_added_to_hass(self) -> None:
                     attr_path_filter=attr_path,
                 )
             )
+        await self.matter_client.subscribe_attribute(
```

Comment:
it's a bit weird that we call `subscribe` and then call `subscribe_attribute`. Which one does the actual subscription? And is there a reason why we need to pass the `sub_paths` again? Couldn't we just look into `matter_client` and find all attr_path_filters ? 

(If valid concern, this comment can be addressed in a future PR)

Reply:
You setup a subscription for events and then you receive node events (including attribute changes) 
but you only receive attribute changes for the attributes you explictly subscribe to 
and we do that from the entity model where we use all attributes
the attributes we actually need are derrived from the discovery schemas
and we do a quick check if the node/endpoint has the attribute